### PR TITLE
Homepage Hero Section

### LINKED ISSUE #33: Create Homepage Hero Component
Create styled Hero component for use as homepage aside.
[For reference](https://css-tricks.com/creating-non-rectangular-headers/)

Preview:
smartshopperr-homepage

### PR DESCRIPTION
**Pull request by sanrafa**
## Description

This PR adds a hero section to the homepage: a curved header with a gradient background and picture logo superimposed.

I achieved this by drawing the shape in Figma, exporting and saving its SVG code, then setting that as the `mask-image` on the hero element; this carves the shape out of the gradient background. 

The page is responsive, width under 768px has pure gradient background.

Also, moved `Home.css` to the src/pages folder.

## Related Issue

Closes #33 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |

### Before

header-before

### After

Desktop preview:
hero-desktop

Mobile preview:
hero-mobile

### CODE REVIEW

**Review by vividavila98 — APPROVED** [review]
Rafa it looks SOO good!! I love the curved shape and the responsiveness is so clean. :))

I have a very small nitpick that doesn't make a huge difference so please take it or leave it!

It looks like the circle in the graphic is cut off a little from the top. I'm thinking in Figma it needs to be resized to show the whole circle: 

image

Again, not a big deal, and the page still looks amazing!! Thanks for taking care of this harder design :)

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -18,19 +18,22 @@ export const Home = (props) => {
...
+      <main>
+        <h1>Welcome to your Smart Shopping List!</h1>
+        {token ? (
+          <OrangeButton>
+            <Link to="/list">View your shopping list</Link>
+          </OrangeButton>
+        ) : (
           <OrangeButton onClick={() => generateToken()}>
-            Create new List
+            Create a new list
```
Comment (danainjax) [inline src/pages/Home.jsx:32]: Thanks for updating this. I was working off exactly what was on the wireframe. I should have used a more holistic approach.

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -18,19 +18,22 @@ export const Home = (props) => {
...
-      {token ? (
-        <button>
-          <Link to="/list">View your shopping list</Link>
-        </button>
-      ) : (
-        <>
+      <header className="hero">
+        <img src={HeroImage} alt="Logo" />
+      </header>
+      <main>
```
Comment (danainjax) [inline src/pages/Home.jsx:24]: Getting semantic. 😃

File: src/pages/Home.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+@media screen and (max-width: 768px) {
+  .home {
+    background-image: linear-gradient(
+      205.21deg,
+      #76dd78 0.82%,
+      rgba(118, 221, 120, 0) 112.11%
+    );
+    justify-content: center;
+  }
```
Comment (danainjax) [inline src/pages/Home.css:47]: Very nice responsiveness!

File: src/assets/hero.svg [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+              y1="-3.51209e-05"
+              x2="6.45975e-06"
+              y2="671.5"
+              gradientUnits="userSpaceOnUse"
+            >
+              <stop stopColor="#76DD78" />
+              <stop offset="1" stopColor="#76DD78" stopOpacity="0" />
+            </linearGradient>
+          </defs>
+        </svg>
```
Comment (danainjax) [inline src/assets/hero.svg:24]: I don't know how this works, but I'd love to learn.

**Review by danainjax — APPROVED** [review]
It just looks so great, Rafa
Screen Shot 2022-02-28 at 11 42 10 AM
!

File: src/assets/hero.svg (hunk 4 above)
Comment (sanrafa) [reply src/assets/hero.svg:24]: To be honest, I don't quite understand SVG yet either, haha. I just redrew the shape in Figma and copy/pasted the SVG code into a new file.

**Pull request merged**